OddJobBoard/api.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework import serializers
+from rest_framework import serializers, viewsets
+from .models import PersonalNote
+
```

Comment:
This is set up correctly for Notes but for the OddJobBoard you will need to change the names of everything in this file.